e2e/vertical-full-page-map.spec.ts

```diff
@@ -91,28 +78,24 @@ test.describe('full page map with filters test suite', () => {
 
   test('clicking on a pin closes the filter view', async ({ page }) => {
     await page.getByRole('button', { name: 'filter results' }).click();
-    await page.getByText('Cats (1)').click();
-    const filterView = page.getByLabel('Main location search').locator('div').filter({ hasText: 'Filters Services reset Cats (1) Dogs (1) Sleep (1)' }).first();
-    await expect(filterView).toBeVisible();
-    await page.getByRole('button', { name: 'Result number 1' }).click();
-    await expect(filterView).not.toBeVisible();
+    const filters = page.locator('#js-answersFacets');
+    await expect(filters).toBeVisible();
+    await page.getByRole('button', { name: 'Result number 7' }).click();
+    await expect(filters).not.toBeVisible();
   });
 
-  test('clicking on a cluster causes the map to zoom in', async ({ page }) => {
+  test('clicking on a cluster causes the map to zoom in and new search is ran', async ({ page }) => {
+    const result = page.locator('#js-answersVerticalResults div').nth(0);
+    await expect(result).toBeAttached();
```

Comment:
is this check needed for timing purposes? if so, it might need to be wrapped in a `waitFor`. same with below (line 97)